cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -182,18 +182,19 @@ handleSimpleNode p trace nodeTracers cc = do
 
         runNetworkArgs :: RunNetworkArgs Peer blk
         runNetworkArgs = RunNetworkArgs
-          { rnaIpSubscriptionTracer  = ipSubscriptionTracer  nodeTracers
-          , rnaDnsSubscriptionTracer = dnsSubscriptionTracer nodeTracers
-          , rnaDnsResolverTracer     = dnsResolverTracer     nodeTracers
-          , rnaMuxTracer             = muxTracer             nodeTracers
-          , rnaMuxLocalTracer        = nullTracer
-          , rnaMkPeer                = Peer
-          , rnaMyAddrs               = addrs
-          , rnaMyLocalAddr           = myLocalAddr
-          , rnaIpProducers           = ipProducers
-          , rnaDnsProducers          = dnsProducers
-          , rnaHandshakeTracer       = nullTracer
-          , rnaHandshakeLocalTracer  = nullTracer
+          { rnaIpSubscriptionTracer    = ipSubscriptionTracer  nodeTracers
+          , rnaDnsSubscriptionTracer   = dnsSubscriptionTracer nodeTracers
+          , rnaDnsResolverTracer       = dnsResolverTracer     nodeTracers
+          , rnaMuxTracer               = muxTracer             nodeTracers
+          , rnaMuxLocalTracer          = nullTracer
+          , rnaMkPeer                  = Peer
+          , rnaMyAddrs                 = addrs
+          , rnaMyLocalAddr             = myLocalAddr
+          , rnaIpProducers             = ipProducers
+          , rnaDnsProducers            = dnsProducers
+          , rnaHandshakeTracer         = nullTracer
+          , rnaHandshakeLocalTracer    = nullTracer
```

Comment:
There are some extra unnecessary spaces before the `=`. These changes can be undone, only the line below needs to be added.